### ISSUE TITLE
Changing the default background from white to any color

### ISSUE THREAD

**Issue by j0hnw0rk3r** [mnmxmx/fluid-three#3]
Hi, I'm looking to implement this project into my website but I need it to adapt to my website's body background color. Can you please help?

**Comment by ardeshiir** [comment]
@j0hnw0rk3r 
I highly recommend reading about fragments and vertices in the context of shaders, but the short version is shaders are made out of two main parts, vertex and fragment, you may think of vertex as the geometry of the shape that is being rendered, and think of fragment as the skin or the color of the shape, so if we want to change the color, we have to look for the right .frag file, in your case (background color), you have to change to color in color.frag file, and remember that colors values in fragment are in range of 0-1 not 0-255.
TL;DR
change `    color = mix(vec3(1.0), color, len);`  to  `    color = mix(vec3(0), color, len);` in color.frag file.
I hope this will help you in some way.
